src/buildercore/command.py

```diff
@@ -47,15 +80,30 @@ def settings(*args, **kwargs):
 
     return fab_api.settings(*args, **kwargs)
 
-lcd = fab_api.lcd # local change dir
-rcd = fab_api.cd # remote change dir
+#
+
+env = api(fab_api.env, threadbare.state.ENV)
+
+local = api(fab_api_local_wrapper, threadbare.operations.local)
+execute = api(fab_api.execute, threadbare.execute.execute_with_hosts)
+parallel = api(fab_api.parallel, threadbare.execute.parallel)
+serial = api(fab_api.serial, threadbare.execute.serial)
+
+hide = api(fab_api.hide, threadbare.operations.hide)
+
+settings = api(fab_api_settings_wrapper, threadbare.state.settings)
+
+lcd = api(fab_api.lcd, threadbare.operations.lcd) # local change dir
+rcd = api(fab_api.cd, threadbare.operations.rcd) # remote change dir
+
+remote = api(fab_api_run_wrapper, threadbare.operations.remote)
+remote_sudo = api(fab_api_sudo_wrapper, threadbare.operations.remote_sudo)
+upload = api(fab_api.put, threadbare.operations.upload)
+download = api(fab_api.get, threadbare.operations.download)
+remote_file_exists = api(fab_files.exists, threadbare.operations.remote_file_exists)
 
-remote = fab_api.run
-remote_sudo = fab_api.sudo
-upload = fab_api.put
-download = fab_api.get
-remote_file_exists = fab_files.exists
-network_disconnect_all = fabric.network.disconnect_all
+network_disconnect_all = api(fabric.network.disconnect_all,
+                             no_match("threadbare automatically closes ssh closes connections"))
```

Comment:
would call this `no_op()` as the implementation does nothing (there could be one that throws an exception for example)

Reply:
renamed